packages/lavamoat-protect/lib/setup-scripts.js

```diff
@@ -1,40 +1,24 @@
-const {writeRcFile} = require('@lavamoat/allow-scripts/setup')
+const promptly = require('promptly')
+const {
+  patchPackageJson,
+  writeRcFile,
+} = require('@lavamoat/allow-scripts/setup')
 const {
   getPackageJson,
-  writePackageJson,
 } = require('./utils')
-const promptly = require('promptly')
-module.exports = async function setupScripts(pkgManager){
-  const pkgConf = getPackageJson()
-  if (pkgConf.scripts) {
-    // TODO: check conflicts, existing stuff?
-  }
-
-  if(!pkgConf.scripts) {
-    pkgConf.scripts = {}
-  }
 
-  if (pkgConf.scripts['lavamoat-postinstall']) {    
+module.exports = async function setupScripts(pkgManager) {
+  const pkgConf = getPackageJson()
+  if (pkgConf.scripts['lavamoat-postinstall']) {
     console.warn('Existing script `lavamoat-postinstall` detected. This is indicative of inconsistently configured LavaMoat')
-    
+
     if(!(await promptly.confirm('Would you like to override the existing script?'))) {
       throw new Error('User exit');
     }
   }
 
-  pkgConf.scripts['lavamoat-postinstall'] = './node_modules/@lavamoat/allow-scripts/src/cli.js'
+  patchPackageJson(pkgManager)
```

Comment:
if patchPackageJson can run pkg commands, we'd need it to be after the RC file is crated.

Reply:
Updated